loghub/main.py

```diff
@@ -164,7 +166,7 @@ def format_changelog(repo,
         version=version, date=close_date, q=quotes)
 
     lines.append(header)
-    lines.append('### Bugs fixed\n')
+    lines.append('### Issues closed\n')
```

Comment:
@ccordoba12 I mean that the code currently also displays PRs that were closed (due to errors or something?) we only want the ones that actually got merged.